.vscode/settings.json

```diff
@@ -1,5 +0,0 @@
-{
-    "typescript.tsdk": "node_modules/typescript/lib",
-    "editor.formatOnSave": true,
```

Comment:
:(

Reply:
Wait why would you need my copy of this file I'm confused

Reply:
I don't think it's a bad idea to "force" others to make prettier work when they save. They might not have it enabled already. I also somehow lose it entirely so I can recreate it, nbd.

This could be an enhancement in another PR for the readme to tell about devs this useful file and settings.